### ISSUE TITLE
Error With Python 3.8 but Not With Python 3.9

### ISSUE THREAD

**Issue by jmwright** [CadQuery/cadquery#796]
## To Reproduce

A CadQuery environment created with the following line works:

```
conda install -c cadquery -c conda-forge cadquery=master python=3.9
```

But this does not create a working environment.

```
conda install -c cadquery -c conda-forge cadquery=master python=3.8
```

## Backtrace

This is what I get if I try to import cadquery in the Python 3.8 environment.

```
$ python
Python 3.8.10 | packaged by conda-forge | (default, May 11 2021, 07:01:05) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cadquery
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jwright/Downloads/cadquery/__init__.py", line 2, in <module>
    from .occ_impl.geom import Plane, BoundBox, Vector, Matrix, Location
  File "/home/jwright/Downloads/cadquery/occ_impl/geom.py", line 5, in <module>
    from OCP.gp import gp_Vec, gp_Ax1, gp_Ax3, gp_Pnt, gp_Dir, gp_Trsf, gp_GTrsf, gp, gp_XYZ
ImportError: /home/jwright/anaconda3/envs/cadquery2/lib/python3.8/site-packages/OCP.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN18BRepOffset_Inter2d14ConnexIntByIntERK11TopoDS_FaceR17BRepOffset_OffsetR19NCollection_DataMapI12TopoDS_ShapeS6_23TopTools_ShapeMapHasherERKS8_RKN11opencascade6handleI14BRepAlgo_AsDesEEddRK18BRepOffset_AnalyseR22NCollection_IndexedMapIS6_S7_ER26NCollection_IndexedDataMapIS6_16NCollection_ListIS6_ES7_E
```

## Environment

OS: Ubuntu 18.04
Conda Version: 4.8.5

**Comment by adam-urbanczyk** [comment]
Is it fixed by #790?

**Comment by jmwright** [comment]
Yes, sorry for the duplicate.
